<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editWindow</class>
 <widget class="QDialog" name="editWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>370</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>530</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Script MT</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit A Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>309</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Resource's ID</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ID1 (Type)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="oldid1">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="oldBook">
           <property name="text">
            <string>Book (0)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="oldArticle">
           <property name="text">
            <string>Article (1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="oldDigitalContent">
           <property name="text">
            <string>Digital Content (2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="oldOther">
           <property name="text">
            <string>Other (3)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ID2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="oldId2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>76</y>
     <width>481</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Check The Properties You Want to Edit and Fill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string> Them With The New Information</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QCheckBox" name="isType">
        <property name="text">
         <string>Change Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="newType">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="newBook">
           <property name="text">
            <string>Book</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="newArticle">
           <property name="text">
            <string>Article</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="newDigitalContent">
           <property name="text">
            <string>Digital Content</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="newOther">
           <property name="text">
            <string>Other</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QCheckBox" name="isTitle">
        <property name="text">
         <string>Change Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newTitle"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="isAuthor">
        <property name="text">
         <string>Change Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newAuthor"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QCheckBox" name="isPublicationYear">
        <property name="text">
         <string>Change Publication Year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="newPublicationYear"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QCheckBox" name="isQuantity">
        <property name="text">
         <string>Change Quantity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="newQuantity"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit The Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset The Enterred Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>580</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go Back to The Main Window</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
